Purchases/Classes/Public/RCPurchases.m

```diff
@@ -41,16 +41,60 @@ @interface RCPurchases () <RCStoreKitWrapperDelegate>
 
 @implementation RCPurchases
 
+static RCPurchases *_singleton = nil;
```

Comment:
`_sharedInstance` is a more typical name, or `_defaultInstance`.

Actually, what do you think about `sharedPurchases`? That would match what Segment does.